### PR TITLE
feat(paddle): Add backward method to Paddle Frontend

### LINKED ISSUE #23407: backward

### PR DESCRIPTION
**Pull request by Boghdady9**
# PR Description

Added backward method to Paddle Frontend

## Related Issue

Close #23407

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by Boghdady9** [comment]
@AnnaTz 

Please review my PR

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -747,3 +747,9 @@ def remainder(self, y, name=None):
 
     def is_floating_point(self):
         return paddle_frontend.is_floating_point(self._ivy_array)
+
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle")
+
```
Comment (AnnaTz) [inline ivy/functional/frontends/paddle/tensor/tensor.py:753]: Hi @Boghdady9, please resolve the lint errors found in the CI checks at the bottom of the PR.

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 2]
```diff
@@ -747,3 +747,9 @@ def remainder(self, y, name=None):
...
+    def backward(self,x,grad_tensor=None,retain_graph=False,):
```
Comment (AnnaTz) [inline ivy/functional/frontends/paddle/tensor/tensor.py:754]: Also, you will need to add a test for the function you implemented. Feel free to read through our [docs](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html#ivy-frontend-tests) and use the rest of the existing tests as reference.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by Boghdady9** [comment]
Hi @AnnaTz 
I just need to add instance methods  the paddle.tensor.tensor.Tensor class. I not asked to add test function to it. check that : https://github.com/unifyai/ivy/issues/15115

**Comment by AnnaTz** [comment]
ivy-gardener

**Pull request closed without merging**